docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.md

```diff
@@ -17,18 +17,17 @@ To start feature integration, integrate the required features:
 
 | NAME | VERSION | INTEGRATION GUIDE |
 | --------- | ------ | ---------------|
-| Spryker Core | 202001.0 | [Spryker Core feature integration](https://documentation.spryker.com/docs/spryker-core-feature-integration) |
```

Comment:
please revert Spryker Core

Reply:
not a problem, but why we need to leave it since `Order Management` depends on it. So, it is redundant.